stackrox/collector
# ROX-372: Scrape connections from proc

To test:
- build collector locally
- run collector with the required capabilities and mounts:  
`$ docker run -it --cap-add SYS_PTRACE -v /proc:/host/proc --entrypoint /bin/bash stackrox/collector:latest`
- Compare the output of `connscrape /proc` and `connscrape /host/proc`. The former should only list active connections in the current container (you can do, e.g., `apt-get update` and hit Ctrl+Z while it's downloading something), while the latter should list active connections in all containers running on the host.
